package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teamleader/ui-icons",
   "description": "Teamleader UI icons",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
**Strong PP:** Version should be `1.2.0` since it's an addition, not a patch.

Reply:
Aha, ok thanks will change 👍